frontend/src/components/FormProgress/index.tsx

```diff
@@ -11,56 +12,64 @@ import Subtitle from '../Subtitle';
 
 import './styles.scss';
 
-const FormProgress = () => (
-  <div className="form-progress">
-    <div className="form-progress-title-section">
-      <p className="form-progress-title">
-        Form progress
-      </p>
-      <Subtitle text="Where you are in the registration process" />
-    </div>
-    <div className="steps-box">
-      <ProgressIndicator
-        currentIndex={5}
-        spaceEquall
-      >
-        <ProgressStep
-          label="Collection"
-          secondaryLabel="Step 1"
-        />
-        <ProgressStep
-          label="Ownership"
-          secondaryLabel="Step 2"
-        />
-        <ProgressStep
-          label="Interim storage"
-          secondaryLabel="Step 3"
-        />
-        <ProgressStep
-          label="Orchard"
-          secondaryLabel="Step 4"
-        />
-        <ProgressStep
-          label="Parent tree and SMP"
-          secondaryLabel="Step 5"
-        />
-        <ProgressStep
-          label="Extraction and storage"
-          secondaryLabel="Step 6"
-        />
-      </ProgressIndicator>
-    </div>
-    <div>
-      <Button
-        kind="tertiary"
-        size="md"
-        className="btn-fp"
-        renderIcon={DocumentBlank}
-      >
-        Complete registration
-      </Button>
+interface FormProgressProps {
+  seedlotNumber: number;
+}
+
+const FormProgress = ({ seedlotNumber }: FormProgressProps) => {
+  const navigate = useNavigate();
+  return (
+    <div className="form-progress">
+      <div className="form-progress-title-section">
+        <p className="form-progress-title">
+          Form progress
+        </p>
+        <Subtitle text="Where you are in the registration process" />
+      </div>
+      <div className="steps-box">
+        <ProgressIndicator
+          currentIndex={5}
+          spaceEquall
```

Comment:
I get a warning in my Chrome browser about the `spaceEquall`, did you mean `spaceEqually`? 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/25162561/226219180-bbf0b542-9f84-4430-bb60-04d2b13fa2c1.png">

Reply:
this warning does not show on the page after clicking into `Complete registration`

Reply:
> this warning does not show on the page after clicking into `Complete registration`

this will not happen if both pages are sharing the same component 😉

Reply:
Yep, I agree, I created a new component for seedlot's form progress, it's still focused only at seedlots, if one day we need it for other cases, the change will be minimal.